src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/ProductPreferences/GeneralType.php

```diff
@@ -64,7 +65,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'required' => true,
             ])
             ->add('force_friendly_url', SwitchType::class)
-            ->add('default_status', SwitchType::class);
+            ->add('default_status', SwitchType::class)
+            ->add('specific_price_priorities', PriorityListType::class, [
+                'required' => false,
+                'label' => false,
```

Comment:
If so it's worth migrating this form Though it might make this PR a bit too big so I would suggest creating a dedicated PR that will only add this new field in the configuration form And this PR will only deal with the product form once the other is merged